src/CircularProgress.js

```diff
@@ -33,10 +26,10 @@ export default class CircularProgress extends React.Component {
 
   render() {
     const { size, width, tintColor, backgroundColor, style, rotation, linecap, children } = this.props;
-    const backgroundPath = this.circlePath(size / 2, size / 2, size / 2 - width / 2, 0, 360);
+    const backgroundPath = this.circlePath(size / 2, size / 2, size / 2 - width / 2, 0, 360 * 99.9 / 100);
```

Comment:
You have used `* .99` later in the same file, here you use `* 99.9 / 100`. If we really need to do this, even though it feels hacky to me, we should stick to either and not use both.